### PR TITLE
Ingredients improvements for various European languages

### PR DESCRIPTION
**Pull request by teolemon**
Ingredients improvements for various European languages

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -21439,14 +21566,18 @@ es:Sabor a cebolla
 fi:sipuliaromi
 fr:Arôme d'oignon
 hu:hagyma aroma
+it:Aroma di cipolla
+nl:uiensmaak
```
Comment (aleene) [inline taxonomies/ingredients.txt:21570]: `uiensmaak` can be deleted, should be `uienaroma`.

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -22683,7 +22835,6 @@ it:fiocchi di frumento
 nb:hveteflak
 nl:tarwevlokken
 pl:płatki pszenne
-ro:fulgi de grâu
```
Comment (aleene) [inline taxonomies/ingredients.txt:22686]: I think this is ok. https://www.sanovita.ro/fulgi-de-grau-500g-174.html

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -23002,14 +23155,14 @@ nb:hvetekim
 nl:tarwekiem
 pl:kielki pszenicy
 pt:gérmen de trigo
-ro:germeni de grâu
```
Comment (aleene) [inline taxonomies/ingredients.txt:23005]: should be `germenii de grâu`

File: taxonomies/ingredients.txt [hunk 4]
```diff
@@ -23454,7 +23608,6 @@ hu:árpapehely
 it:fiocchi d'orzo
 nl:gerstvlokken
 pl:płatki jęczmienne
-ro:fulgi de orz
```
Comment (aleene) [inline taxonomies/ingredients.txt:23457]: is ok

File: taxonomies/ingredients.txt [hunk 5]
```diff
@@ -23993,7 +24150,7 @@ nb:havreflak, havregryn
 nl:havermoutvlokken, havermout, havervlokken, havervlok
 pl:płatki owsiane
 pt:flocos de aveia
-ro:fulgi de ovăz, Haferflocken
+ro:Haferflocken
 ru:Овсяные хлопья
```
Comment (aleene) [inline taxonomies/ingredients.txt:24154]: mixing romanian and german. keep the deleted version.

File: taxonomies/ingredients.txt [hunk 6]
```diff
@@ -24034,7 +24191,6 @@ hu:zabrost
 it:fibra d'avena
 nl:havervezel
 pt:fibra de aveia
-ro:fibre de ovăz
```
Comment (aleene) [inline taxonomies/ingredients.txt:24037]: is ok

File: taxonomies/ingredients.txt [hunk 7]
```diff
@@ -24381,7 +24547,6 @@ fr:flocons de seigle
 hu:rozspehely
 it:fiocchi di segale
 nb:rugflak
-ro:fulgi de secară
```
Comment (aleene) [inline taxonomies/ingredients.txt:24384]: is ok

File: taxonomies/ingredients.txt [hunk 8]
```diff
@@ -25000,10 +25171,6 @@ fr:farine de blé dextrinée
 hu:dextrinált búzaliszt
 nl:gedextrineerd tarwemeel
 
-<en:wheat flour
-en:pre-gelatinized wheat flour
-fr:farine de blé prégélatinisée, farine de froment prégélatinisée
-
```
Comment (aleene) [inline taxonomies/ingredients.txt:25006]: what is wrong with this.

File: taxonomies/ingredients.txt [hunk 9]
```diff
@@ -24988,8 +25159,8 @@ fr:farine de blé complète, farine de blé complet, farine complète de blé, f
 hu:teljes kiőrlélsű búzaliszt
 it:farina integrale di frumento, farina di frumento integrale
 nl:volkorentarwemeel, volkoren tarwemeel
+pl:Mąka pszenna pełnoziarnista
 pt:farinha de trigo integral, farinha integral de trigo
-ro:făină integrală de grâu
```
Comment (aleene) [inline taxonomies/ingredients.txt:24992]: probably ok

File: taxonomies/ingredients.txt [hunk 10]
```diff
@@ -25251,13 +25422,10 @@ da:sigtet rugmel
 fi:lestyruisjauho, ruissihtijauho
 sv:siktat rågmjöl, rågsiktmjöl
 
-<en:rye flour
-en:pregelatinized rye flour
-ro:făină de secară pregelatinizată
-
```
Comment (aleene) [inline taxonomies/ingredients.txt:25257]: is ok

File: taxonomies/ingredients.txt [hunk 11]
```diff
@@ -25459,10 +25635,6 @@ fr:farine de riz dextrinée
 hu:dextrinált rizsliszt
 nl:gedextrineerd rijstmeel
 
-<en:rice flour
-en:pregelatinized rice flour
-fr:farine de riz prégélatinisée
-
```
Comment (aleene) [inline taxonomies/ingredients.txt:25465]: ?

File: taxonomies/ingredients.txt [hunk 12]
```diff
@@ -25473,13 +25645,14 @@ es:harina de arroz integral
 fi:tumma riisijauho, täysjyväriisijauho
 fr:farine de riz complet, farine complète de riz
 hu:barnariszliszt
+it:Farina di riso nero
 nl:volkoren rijstmeel
 
 <en:rice flour
 <en:wholemeal flour
 en:whole rice flour, whole grain rice flour
-de:Vollkornreismehl
```
Comment (aleene) [inline taxonomies/ingredients.txt:25481]: is ok?

File: taxonomies/ingredients.txt [hunk 13]
```diff
@@ -27633,11 +27837,8 @@ it:zucchero d'uva
 nb:druesukker
 sv:druvsocker
 
-# description:en:Re vanilla sugar: "Cheaper vanilla sugar is also available, made only from sugar and vanillin."
 <en:sugar
-<en:vanillin
 fr:sucre vanilline
-de:Vanillinzucker
 fi:vanilliinisokeri
```
Comment (aleene) [inline taxonomies/ingredients.txt:27842]: german translation is ok

File: taxonomies/ingredients.txt [hunk 14]
```diff
@@ -28356,7 +28558,6 @@ fr:oligofructose
 it:oligofruttosio
 nb:oligofruktose
 nl:oligofructose
-ro:oligofructoză
```
Comment (aleene) [inline taxonomies/ingredients.txt:28359]: why delete?

File: taxonomies/ingredients.txt [hunk 15]
```diff
@@ -28752,13 +28966,6 @@ nl:dextrose van tarwe of maïs
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/wheat-or-corn-dextrose
 # 64 products in 3 languages @2018-09-25
 
-<en:dextrose
-en:dextrose monohydrate
-es:dextrosa monohidrato, dextrosa monohidratada
-fr:dextrose monohydraté
-nl:dextrose monohydraat
-pt:dextrose monohidratada
-
```
Comment (aleene) [inline taxonomies/ingredients.txt:28761]: Is in principle OK. can be synonym of dextrose.

File: taxonomies/ingredients.txt [hunk 16]
```diff
@@ -29735,8 +29962,9 @@ fr:beurre de cacao émulsifiant
 
 <en:cocoa
 en:cocoa fat
+de:Kakaobutter
 es:grasa de cacao
-ro:grăsime de cacao
+nl:kokosvet
```
Comment (aleene) [inline taxonomies/ingredients.txt:29967]: kokosvet is wrong, should be cacaovet

File: taxonomies/ingredients.txt [hunk 17]
```diff
@@ -29735,8 +29962,9 @@ fr:beurre de cacao émulsifiant
 
 <en:cocoa
 en:cocoa fat
+de:Kakaobutter
```
Comment (aleene) [inline taxonomies/ingredients.txt:29965]: Kakaobutter  this should be under cocoa butter

File: taxonomies/ingredients.txt [hunk 18]
```diff
@@ -29735,8 +29962,9 @@ fr:beurre de cacao émulsifiant
...
 es:grasa de cacao
-ro:grăsime de cacao
```
Comment (aleene) [inline taxonomies/ingredients.txt:29739]: looks ok

File: taxonomies/ingredients.txt [hunk 19]
```diff
@@ -30302,7 +30537,6 @@ nl:Melkchocola, melkchocolade
 nn:Mjølkesjokolade
 pl:Czekolada mleczna
 pt:chocolate de leite, chocolate ao leite
-ro:ciocolată cu lapte
```
Comment (aleene) [inline taxonomies/ingredients.txt:30305]: ok

File: taxonomies/ingredients.txt [hunk 20]
```diff
@@ -30936,6 +31176,7 @@ sv:Filterat vatten
 
 <en:water
 en:cooking water
+de:kochendes Wasser
```
Comment (aleene) [inline taxonomies/ingredients.txt:31179]: Does not sound ok, should be `Kochwasser`, kochendes wasser is water that is cooking (now)

File: taxonomies/ingredients.txt [hunk 21]
```diff
@@ -31383,12 +31626,12 @@ es:gluten de trigo
 et:nisugluteen
 fi:vehnägluteeni
 fr:gluten de blé, gluten de froment
+hu:Búzaglutén
 it:glutine di grano, glutine di frumento
 nb:hvetegluten
 nl:tarwegluten
 pl:gluten pszenny
 pt:glúten de trigo
-ro:gluten de grâu, gluten din grâu
```
Comment (aleene) [inline taxonomies/ingredients.txt:31391]: ok?

File: taxonomies/ingredients.txt [hunk 22]
```diff
@@ -33750,12 +34009,12 @@ fi:kuivattu banaani, kuivatut banaanit
 fr:bananes séchées
 it:banane secche
 nl:gedroogde banaan
-ro:banane uscate
```
Comment (aleene) [inline taxonomies/ingredients.txt:33753]: ?

File: taxonomies/ingredients.txt [hunk 23]
```diff
@@ -35062,11 +35335,10 @@ fr:pur jus de pommes à cidre
 # ingredient/fr:pur-jus-de-pommes-a-cidre has 10 products @2019-05-28
 
 <en:cidre apple juice
-en:organic cidre apple juice
-fr:pur jus de pommes à cidre bio
 <en:apple
```
Comment (aleene) [inline taxonomies/ingredients.txt:35338]: is ok

File: taxonomies/ingredients.txt [hunk 24]
```diff
@@ -35062,11 +35335,10 @@ fr:pur jus de pommes à cidre
...
-en:Golden Delicious apple
+en:organic cidre apple juice
```
Comment (aleene) [inline taxonomies/ingredients.txt:35339]: what is happening here? wrong deletes.

File: taxonomies/ingredients.txt [hunk 25]
```diff
@@ -37519,7 +37812,7 @@ nl:dadels
 nn:dadler
 nv:Hashkʼaan dijéʼé
 pl:Daktyle
-ro:Curmală, curmale
+ro:Curmală
 ru:финик, финики
```
Comment (aleene) [inline taxonomies/ingredients.txt:37816]: the synonym should be kept. Has been found on products.

File: taxonomies/ingredients.txt [hunk 26]
```diff
@@ -39062,6 +39370,7 @@ sv:frystorkade hallon
 en:raspberry powder
 de:Himbeerpulver, Himbeerfruchtpulver
 fi:vadelmajauhe
+fr:Poudre de fraise
```
Comment (aleene) [inline taxonomies/ingredients.txt:39373]: should appear under strawberry powder

File: taxonomies/ingredients.txt [hunk 27]
```diff
@@ -39953,10 +40279,11 @@ fi:puristemehu, rypäleen puristemehu
 fr:moût de raisin
 it:mosto d'uva
 nl:druivenmost
-ro:must de struguri
```
Comment (aleene) [inline taxonomies/ingredients.txt:39956]: is ok

**Review by aleene — CHANGES_REQUESTED** [review]
All the romanian deletes seem to be ok (checked with a translator).

**Comment by svensven** [comment]
> All the romanian deletes seem to be ok (checked with a translator).

I think a lot of the deletions are (presumably unintentional) reverts of things I've added recently.

File: taxonomies/ingredients.txt [hunk 28]
```diff
@@ -1510,6 +1516,7 @@ fr:pectine de fruit, pectines de fruits
 hu:gyümölcspektin
 it:pectina di frutta
 nl:vruchtenpectine, fruitpectine
+pl:Pektyna, pektyna owocowa, pektyna z owoców
```
Comment (teolemon) [inline taxonomies/ingredients.txt:1519]:
```suggestion
pl:Pektyna owocowa, pektyna z owoców
```

File: taxonomies/ingredients.txt [hunk 29]
```diff
@@ -5207,6 +5216,7 @@ nl:skyr-culturen
 <en:lactic ferments
 en:sourdough starter, sourdough culture
 da:surdejskultur
+fr:Starters pour levain
```
Comment (teolemon) [inline taxonomies/ingredients.txt:5219]:
```suggestion
fr:Préparations pour levain, Starters pour levain
```

[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/ingredients.txt:5219]: I took that from real-life examples in the DB

File: taxonomies/ingredients.txt [hunk 30]
```diff
@@ -5987,7 +5998,6 @@ wikipedia:en:https://en.wikipedia.org/wiki/Cellulase
 # ingredient/fr:cellulase has 173 products in 6 languages @2019–07-09
 
 <en:cellulase
```
Comment (teolemon) [inline taxonomies/ingredients.txt:6000]:
```suggestion
<en:cellulase
en:hemicellulase
```

File: taxonomies/ingredients.txt [hunk 31]
```diff
@@ -6787,7 +6798,9 @@ fr:lait thermisé
 
 <en:milk
 en:milk solids
+de:Milchpulver
```
Comment (teolemon) [inline taxonomies/ingredients.txt:6801]:
```suggestion
```

File: taxonomies/ingredients.txt [hunk 32]
```diff
@@ -7723,7 +7741,6 @@ it:latte in polvere
 nl:melkpoeder
 pl:mleko w proszku
 pt:leite em pó
```
Comment (teolemon) [inline taxonomies/ingredients.txt:7743]:
```suggestion
pt:leite em pó
ro:lapte praf
```

**Comment by teolemon** [comment]
This is stale. We can recover translations from the server anyway.

**Pull request closed without merging**